cpp/command/analysis.cpp

```diff
@@ -704,6 +704,7 @@ int MainCmds::analysis(int argc, const char* const* argv) {
         newRequest->hist = hist;
         newRequest->nextPla = nextPla;
         newRequest->params = rbase.params;
+        newRequest->perspective = rbase.perspective;
```

Comment:
This is the only line that needs to be fixed, right?

Reply:
(that is, the other changes are unnecessary, we just need only this single line as the entire commit to fix this bug, right?)

Reply:
you're right, this was the last line I changed before thinking it was the other ones, but the cfg takes care of that. fixed.